packages/indexer/README.md

```diff
@@ -149,7 +149,7 @@ for await (const cell of cellCollector.collect()) {
 }
 ```
 
-You can also set it as `any` when the argsLen of the `args` might have multiple possibilities. However, there's some performance lost when use `any` rather than explicit specify length due to the low-level implementation.
+ You can also set it as any when the argsLen of the field args might have multiple possibilities, for example, lock script's args could be 20 in normal scenario and 28 in multisig scenario,  or any other length in customized scenarios. However, there's some performance lost when use `any` rather than explicit specify length due to the low-level implementation.
```

Comment:
```suggestion
You can also set it as any when the argsLen of the field args might have multiple possibilities, for example, lock script's args could be 20 in normal scenario and 28 in multisig scenario,  or any other length in customized scenarios. However, there's some performance lost when use `any` rather than explicit specify length due to the low-level implementation.
```